pkg/operator/ceph/cluster/osd/config/scheme_test.go

```diff
@@ -158,7 +158,7 @@ func TestMetadataGetPartitionArgs(t *testing.T) {
 }
 
 func TestSchemeEntryGetPartitionArgs(t *testing.T) {
-	e1 := NewPerfSchemeEntry(Bluestore)
+	e1 := NewPerfSchemeEntry("")
```

Comment:
do you think we should have a 3rd scenario in this test function? Empty string, Bluestore, and Filestore.  Empty string and Bluestore could reuse the same verification/assertion code I believe.

Also a comment about why the empty string is tested and what it means could be helpful.

Reply:
Thanks @jbw976 . Added tests for empty, bluestore and filestore. Was not able to re-use the expectedArgs between the two as the code always generates UUIDs for each store scheme object. But otherwise i believe we have it covered.